http/daemon/server.go

```diff
@@ -32,9 +31,6 @@ var (
 func NewRouter() *mux.Router {
 	r := transport.NewAPIRouter()
 
-	r.NewRoute().Methods("POST").Name("GitPushHook").Path("/v9/hook/git").Queries("repo", "{repo}")
```

Comment:
These are useful! At least, the git one -- you can provoke a sync by `curl`ing it, and do so from e.g., a git hook.